clients/admin-ui/src/features/common/custom-fields/hooks.ts

```diff
@@ -136,7 +136,11 @@ export const useCustomFields = ({
 
       // When creating an resource, the fides key may have initially been blank. But by the time the
       // form is submitted it must not be blank (not undefined, not an empty string).
-      const fidesKey = formValues.fides_key || resourceFidesKey;
+      const fidesKey =
+        "fides_key" in formValues && formValues.fides_key !== ""
+          ? formValues.fides_key
+          : resourceFidesKey;
+
```

Comment:
My understanding is custom fields can be added to multiple fides resources (ex: systems, PDs, etc). In the past fides resources were a 1:1 with the fideslang models which used `fides_key` as the id. The new privacy declaration table is a departure from that. It's not a 1:1 match with fideslang so it uses an id instead. 

I added a bit of code that placed the new PD id as a `fides_key` so it would be backwards compatible with the `useCustomField` hook. The reason is the custom fields attached to systems use the `fides_key` as the id.